token/cli/src/main.rs

```diff
@@ -499,77 +515,62 @@ async fn command_set_interest_rate(
 
 async fn command_create_account(
     config: &Config<'_>,
-    token: Pubkey,
+    token_pubkey: Pubkey,
     owner: Pubkey,
     maybe_account: Option<Pubkey>,
+    immutable_owner: bool,
     bulk_signers: Vec<Arc<dyn Signer>>,
 ) -> CommandResult {
-    let minimum_balance_for_rent_exemption = if !config.sign_only {
-        config
-            .program_client
-            .get_minimum_balance_for_rent_exemption(Account::LEN)
-            .await?
-    } else {
-        0
-    };
+    let token = token_client_from_config(config, &token_pubkey);
+    let mut extensions = vec![];
 
-    let mint_info = config.get_mint_info(&token, None).await?;
-    let (account, system_account_ok, instructions) = if let Some(account) = maybe_account {
-        println_display(config, format!("Creating account {}", account));
-        (
-            account,
-            false,
-            vec![
-                system_instruction::create_account(
-                    &config.fee_payer.pubkey(),
-                    &account,
-                    minimum_balance_for_rent_exemption,
-                    Account::LEN as u64,
-                    &mint_info.program_id,
-                ),
-                initialize_account(&mint_info.program_id, &account, &token, &owner)?,
-            ],
-        )
+    let (account, is_associated) = if let Some(account) = maybe_account {
+        (account, false)
```

Comment:
The token client checks to see if it's an associated address when wrapping, how about doing the same thing here?  I only bring this up because you did good work elsewhere :smile: